doc/release-notes/10831-standardize-image-url-of-search-api.md

```diff
@@ -0,0 +1,28 @@
+Search API (/api/search) response will now include new image_url format for the Datafile and Dataverse logo.
+Note to release note writer: this supersedes the release note 10810-search-api-payload-extensions.md
+
+For Dataverse:
+
+- "image_url" (optional)
```

Comment:
If the image_url is always returned, even though the URL returns a 404 for files and a 204 for collections, then it is not an optional field. Anyway, check my review comment, where I suggest not adding the fields to the payload in these cases.